packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js

```diff
@@ -0,0 +1,205 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Autosuggest from 'react-autosuggest'
+import uuid from 'uuid/v4'
+
+import ButtonLink from '../../ButtonLink'
+import {
+  StyledLocationEdit,
+  StyledZipInput
+} from './StyledLocationEdit'
+import IconTocco from '../../IconTocco'
+
+export const getMapsAddress = locationInput => {
+  const mapsBaseAddress = `https://www.google.com/maps/search/?api=1&query=`
+  if (locationInput) {
+    const locationAcc = []
+    for (const key in locationInput) {
+      if (typeof locationInput[key] === 'string') {
+        locationAcc.push(locationInput[key])
+      }
+      if (typeof locationInput[key] === 'object') {
+        locationAcc.push(locationInput[key].display)
+      }
+    }
+    return `${mapsBaseAddress}${locationAcc.join('+')}`
+  }
+  
+  return mapsBaseAddress
+}
+
+export const getSuggestions = (value, suggestions) => {
+  if (suggestions) {
+    const inputValue = value.trim().toLowerCase()
+    const inputLength = inputValue.length
+
+    return inputLength === 0 ? [] : suggestions
+  }
+  return []
+}
+
+export const returnGetSuggestion = attr => suggestion => suggestion[attr]
+
+const renderSuggestion = suggestion => {
```

Comment:
what is the purpose of the `cantonString` variable? You rather just insert the hyphen in the template 

```
  return (
    <div>
      { `${suggestion.zip} .... - ${suggestion.canton}`}
    </div>
  )
```

you additional variable just makes things harder to read.
Any as said in a comment before, you still have to problem that if canton is not given you generate a string like:

> 8006 Zürich - / Switzerland


same for country / countryString